integrationtest/initial_release/shellHelper.d

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+
+    Shell helper functions
+
+    Copyright:
+        Copyright (c) 2017 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module integrationtest.initial_release.shellHelper;
+
+/*******************************************************************************
+
+    Runs a command and returns the output
+
+    Params:
+        pwd = directory to run the command in
```

Comment:
why `p`?

Reply:
I guess because I am used to the shell command `pwd` which stands for `print working directory`